public/app/assetlibrary/item/assetLibraryItemController.js

```diff
@@ -33,14 +33,54 @@
      */
     var getCurrentAsset = function() {
       assetLibraryItemFactory.getAsset(assetId).success(function(asset) {
-        // Ensure that the newest comments are at the top
-        // TODO: Add sorting to allow for replies
-        asset.comments = asset.comments.reverse();
+
+        // Build the comment tree. First, the top level comments
```

Comment:
Can you document that the `comments` array will hold a flat list of comments where each child comment should come after the parent?